modules/UI/UI.js

```diff
@@ -590,6 +590,21 @@ UI.updateUserRole = user => {
 };
 
 /**
+ * Updates the user status.
+ *
+ * @param {JitsiParticipant} user - The user which status we need to update.
+ * @param {string} status - The new status.
+ */
+UI.updateUserStatus = (user, status) => {
+    let displayName = user.getDisplayName();
+    messageHandler.notify(
+        displayName, '', 'connected', "dialOut.statusMessage",
+        {
+            status: UIUtil.escapeHtml(status)
+        });
+};
```

Comment:
Maybe this function should be renamed to specify that it's notifying of user status change.